src/Entity/Conference.php

```diff
@@ -131,6 +131,13 @@ class Conference
      */
     private array $tags = [];
 
+    /**
+     * @ORM\Column(type="jsonb", nullable=true)
+     *
+     * @var array<float>
+     */
+    private ?array $coordinates = [];
```

Comment:
I'm not a big fan of the typing you are using here.
* `?` says the value is nullable, but you never use this feature in the code. I guess it's for legacy data. So let's migrate legacy data from null to `[]`.
* but you initialize data with `[]`.

So now, there is 2 meanings to say data is empty [] and null.
You need to create something consistant.

I would go for : `NULL` in the DB and in the code, and an array not empty when there is something (so inizialize the data here with `null`)